intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: The absolutely essential concepts
+parent: oci-hub
+tags: [intro, OCI, basics]
```

Comment:
Tags need to be pulled from [_topics.yml](https://github.com/oracle-devrel/cool.devo.build/blob/main/_data/topics.yml). Tags that don't appear in that list are ignored.